src/tracing_tool_output.ml

```diff
@@ -110,8 +101,35 @@ module Serve = struct
   ;;
 end
 
+module Share = struct
+  type t = { share_command_filename : string }
+
+  let maybe_param =
+    Option.map Env_vars.share_command_filename ~f:(fun share_command_filename ->
+        let%map_open.Command share = flag "share" no_arg ~doc:"Share trace." in
+        if share then Some { share_command_filename } else None)
+  ;;
+
+  let share_trace_file { share_command_filename } ~store_path =
+    let%map.Deferred.Or_error output =
+      Process.run
+        ~prog:share_command_filename
+        ~args:[ Filename_unix.realpath store_path ]
+        ()
+    in
+    Core.printf "%s%!" output
```

Comment:
This won't show output until the command ends, and it might also swallow stderr? I'd use the cruddy process APIs to transfer stdout/stderr of the subprocess to magic-trace's.